ebuild-maintenance/git/text.xml

```diff
@@ -324,6 +323,98 @@ Closes: https://bugs.gentoo.org/00001
+
+<p>
+Per Gentoo's <uri link="https://www.gentoo.org/glep/glep-0076.html">Copyright
+Policy GLEP</uri>, you must sign-off all your commits to any Gentoo-hosted
+repository with accordance to the 
+<uri link="https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin">
+copyright rules</uri>. 
+</p>
+
+<p>
+When committing work authored by someone else, i.e. a bugzilla patch, or GitHub
+pull request, a sign-off from the original author is always strongly
+recommended to have to indicate the author acknowledges Gentoo's copyright
+policy. It's not however mandatory for every case. Please refer to the example
+table below when determining whether a sign-off from the original author is, or
+is not required.
+</p>
+
+<table>
+  <tr>
+    <ti>
+      a) A contribution with a Signed-off-by line and a known real name.
+    </ti>
+    <ti>
+      =&gt; Can be accepted, because the author confirmed that it is under a free
+      software license. The committer adds another S-o-b line and certifies the
+      commit under point 4 of the 
+      <uri link="https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin">
+      GCO.</uri>
+    </ti>
+  </tr>
+  <tr>
+    <ti>
+      b) A contribution with a S-o-b line and a name known to be a pseudoym.
+    </ti>
+    <ti>
+      =&gt; Same as a), but if it turns out that the contribution was taken from
+      somewhere else and the contribution author doesn't have the right to
+      distribute it under a free software license, you as the committer might
+      get into trouble. So in this situation, do your best to check overlays for
+      matching code, and whether they hold any special copyright claims.
+    </ti>
+  </tr>
+  <tr>
+    <ti>
+      c) A contribution with a S-o-b line and name of unknown status.
+    </ti>
+    <ti>
+      =&gt; Same as a) or b).
+    </ti>
+  </tr>
+  <tr>
+    <ti>
+      d) A contribution without a S-o-b line but not 
+      <uri link="https://www.gnu.org/prep/maintain/html_node/Legally-Significant.html">
+      "legally significant"</uri> (by the FSF's 15-lines rule of thumb).
+    </ti>
+    <ti>
+      =&gt; Can be accepted. The committer adds a S-o-b line and certifies the
+      commit under GCO point 2.
+    </ti>
+  </tr>
+  <tr>
+    <ti>
+      e) A contribution without a S-o-b line and of significant size, but with
+      an independent indication of its license (e.g. copyright and license
+      notices in the file's header).
+    </ti>
+    <ti>
+      =&gt; Can be accepted. The committer adds a S-o-b line and certifies the
+      commit under GCO point 2.
+    </ti>
+  </tr>
+  <tr>
+    <ti>
+      f) A contribution without a S-o-b line and of significant size, without
+      any other indication of its license.
+    </ti>
+    <ti>
+      =&gt; Can <b>not</b> be accepted. There's no indication that the author
+      has released their work under a free license, therefore it must not be
+      distributed by Gentoo.
+    </ti>
+  </tr>
+</table>
```

Comment:
I think we can simplify this to four cases. The first three are very similar, and we don't verify the identity of contributors.

Also a description list fits better than a table, and the item labels and arrows aren't necessary.

```suggestion
<dl>
  <dt>A contribution with a Signed-off-by line by its author</dt>
  <dd>
    Can be accepted, because the author confirmed that it is under a free
    software license. The committer adds another S-o-b line and certifies the
    commit under point 4 of the 
    <uri link="https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin">
    GCO</uri>.
    <note>
    Use common sense here, especially if you don't know the contributor. If the
    contribution was taken from somewhere else and the contributor doesn't have
    the right to distribute it under a free software license, you as the
    committer might get into trouble. So in this situation, do your best to
    check repositories for matching code, and whether they hold any special
    copyright claims.
    </note>
  </dd>
  <dt>
    A contribution without a S-o-b line and of significant size, but with an
    independent indication of its license (e.g. copyright and license notices
    in the file's header)
  </dt>
  <dd>
    Can be accepted. The committer adds a S-o-b line and certifies the commit
    under GCO point 2.
  </dd>
  <dt>
    A contribution without a S-o-b line but not
    <uri link="https://www.gnu.org/prep/maintain/html_node/Legally-Significant.html">
    "legally significant"</uri> (by the FSF's 15-lines rule of thumb)
  </dt>
  <dd>
    Can be accepted. The committer adds a S-o-b line and certifies the commit
    under GCO point 2.
  </dd>
  <dt>
    A contribution without a S-o-b line and of significant size, without any
    other indication of its license
  </dt>
  <dd>
    Can <e>not</e> be accepted. There's no indication that the author has
    released their work under a free license, therefore it must not be
    distributed by Gentoo.
  </dd>
</dl>
```

Reply:
IMHO the table looked better than this. But you can check how it looks now with the dds.